test/e2e/manifest/manifest.go

```diff
@@ -87,6 +91,20 @@ func IngressFromManifest(fileName string) (*extensions.Ingress, error) {
 	return &ing, nil
 }
 
+// IngressToManifest generates a yaml file in the given path with the given ingress.
+// Returns the file name and error, if there was any.
+func IngressToManifest(ing *extensions.Ingress, path string) error {
+	serialized, err := util.MarshalToYaml(ing, extensions.SchemeGroupVersion)
+	if err != nil {
+		return fmt.Errorf("failed to marshal ingress %v to YAML: %v", ing, err)
+	}
+
+	if err := ioutil.WriteFile(path, serialized, 0600); err != nil {
```

Comment:
What guarantees that that the parent directory exists? Is this expected to just error out in that case?

Reply:
Yes, it assumes parent dir to exist. Added a comment for it